dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/AssignedUserQueryParam.java

```diff
@@ -88,6 +89,11 @@ else if ( (mode == null || mode == AssignedUserSelectionMode.PROVIDED)
             this.mode = AssignedUserSelectionMode.PROVIDED;
             this.assignedUsers = assignedUsers;
         }
+        else if ( mode == null )
```

Comment:
Looks correct, but it occurs to me this whole block might be ripe for a facelift. But I think that is out of the scope of this review :)

Reply:
Yes 😅 there might be a cleaner solution. Feel free to give it a try after I merged the PR. I added ample tests. So refactoring should be safe and fun 🎉